content/layouts/page-breadcrumbs-new.html

```diff
@@ -0,0 +1,63 @@
+{% include "content/includes/header.html" %}
+
+{% unless page.template == 'alt-1-topic-landing' %}
+
+{% unless showactionbar == false %}
+
+  {% if path contains 'disability-benefits' %}
+    {% include "content/includes/action-bar.html" action_text="Apply for Disability Benefits" action_url="/disability-benefits/apply-for-benefits/" action_class="va-action-bar--header" %}
+  {% endif %}
+
+  {% if path contains 'education' %}
+    {% include "content/includes/action-bar.html" action_text="Apply for Education Benefits" action_url="/education/apply-for-education-benefits/" action_class="va-action-bar--header" %}
+  {% endif %}
+
+  {% if path contains 'healthcare' %}
+    {% if showtempbar == true %}
+    {% else %}
+      {% include "content/includes/action-bar.html" action_text="Apply for VA Health Care" action_url="/healthcare/apply/application" action_class="va-action-bar--header" %}
+    {% endif %}
+  {% endif %}
+
+{% endunless %}
+
+{% endunless %}
+
+{% include "content/includes/breadcrumbs.html" %}
+
+<div id="content" class="interior {{ id }}">
+
+  {% case template %}
+    {% when 'L2-section-landing' %}
+      {% include "content/includes/L2-section-landing.html" %}
+
+    {% when '1-topic-landing' or 'alt-1-topic-landing' %}
+      {% include "content/includes/1-topic-landing.html" %}
+
+    {% when '2-detailed-landing' %}
+      {% include "content/includes/2-detailed-landing.html" %}
+
+    {% when '3-panel-filters' %}
+       {% include "content/includes/3-panel-filters.html" %}
+
+    {% when '4-action-page' %}
+      {% include "content/includes/4-action-page.html" %}
+
+    {% when '4-action-page-pending' %}
+      {% include "content/includes/4-action-page-pending.html" %}
+
+    {% when '6-info-page' %}
```

Comment:
Yeah, it definitely would work and that would make any edits / merges easier later, but I wasn't as much concerned with copying the code as I was making it as obvious as I could that this is not production-facing until we're ready for it to be.

I could have accomplished that with comments, but I didn't find a way to comment in liquid and I didn't want to put it in HTML comments for fear of it not being stripped away in the production build chain.